src/main/java/edu/wpi/grip/ui/preview/LinesSocketPreviewView.java

```diff
@@ -31,6 +31,7 @@
     private final ImageView imageView = new ImageView();
     private final Label infoLabel = new Label();
     private final Mat tmp = new Mat();
+    private boolean showInputImage = false;
```

Comment:
This isn't true.

@JLLeitschuh Can you disable this bot?  It's kind of annoying.